std_msgs/Bool Forward
---
std_msgs/String imageDir
std_msgs/Int32 nMatches
std_msgs/Int32 nInliers
geometry_msgs/Transform delta